### ISSUE TITLE
Reversed-out gets tiring to look at

### ISSUE THREAD

**Issue by lucymonie** [The-Authenticators/sohapp#55]
I know it's trendy if you're a developer but it gets tiring to look at reversed-out text for page after page. Consider mixing it up a little bit?

**Comment by stevehopkinson** [comment]
This is part of our sweeping plans for Sohapp V2. Alas, currently only in Photoshop form.
![sohapp-mockup](https://cloud.githubusercontent.com/assets/5929394/21387888/b6ffa820-c771-11e6-8d82-844a8e5681aa.jpg)

**Comment by marisid** [comment]
Yep, I'm generally fond of white background with darker font :arrow_right: makes so much difference in readability.

**Comment by stevehopkinson** [comment]
I'm fairly sure I've read a lot of research suggesting that white-on-dark is better. But having googled it a bit, [it looks like I'm completely wrong](http://ux.stackexchange.com/questions/53264/dark-or-white-color-theme-is-better-for-the-eyes). Let's change it!
